src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,3 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="@JAVA_MODULE_NAME@" android:versionCode="1" android:versionName="1.0">
```

Comment:
please add the license header (just like its done in the pom.xml

Reply:
done